release/release.go

```diff
@@ -151,6 +151,62 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client) (map[string]string, error) {
+	const (
+		defaultPerPage int    = 100
+		limitMax       int    = 20
+		owner          string = "kubernetes"
+		repo           string = "kubernetes"
+	)
+	var latestVersion, stableVersion string
+	goVersions := make(map[string]int)
+	maxOccurrences := 0
```

Comment:
We want to take advantage of the copmmunity standard of utilizing the "0 value" for variable declaration. E.g. 

```
var maxOccurances int
```

This is preferred whenever we declare a variable and don't need to set an explicit value.